src/tests/ftest/deployment/ior_per_rank.yaml

```
@@ -0,0 +1,61 @@
+hosts:
+  test_servers: 2
+  test_clients: 2
+timeout: 1500
+server_config:
+    name: daos_server
+    engines_per_host: 2
+    engines:
+    0:
+      pinned_numa_node: 0
+      nr_xs_helpers: 1
+      fabric_iface: ib0
+      fabric_iface_port: 31317
+      log_file: daos_server0.log
+      storage:
+        0:
+          class: dcpm
+          scm_list: ["/dev/pmem0"]
+          scm_mount: /mnt/daos0
+        1:
+          class: nvme
+          bdev_list: ["aaaa:aa:aa.a"]
+    1:
+      pinned_numa_node: 1
+      nr_xs_helpers: 1
+      fabric_iface: ib1
+      fabric_iface_port: 31417
+      log_file: daos_server1.log
+      storage:
+        0:
+          class: dcpm
+          scm_list: ["/dev/pmem1"]
+          scm_mount: /mnt/daos1
+        1:
+          class: nvme
+          bdev_list: ["bbbb:bb:bb.b"]
+pool:
+  mode: 146
+  name: daos_server
+  size: 95%
+  control_method: dmg
+  properties: ec_cell_sz:128KiB
+container:
+    type: POSIX
+    properties: cksum:crc16,cksum_size:16384,srv_cksum:on
+    control_method: daos
+ior:
+    client_processes:
+          ppn: 16
```

For performance, we should use more PPN to make sure we saturate
```suggestion
    client_processes:
          ppn: 32
```

---

Done.